src/types/redis_bitmap.cc

```diff
@@ -219,6 +219,11 @@ rocksdb::Status Bitmap::BitCount(const Slice &user_key, int64_t start, int64_t s
   rocksdb::Status s = GetMetadata(ns_key, &metadata, &raw_value);
   if (!s.ok()) return s.IsNotFound() ? rocksdb::Status::OK() : s;
 
+  /* Convert negative indexes */
```

Comment:
This is same as redis code, don't ask me why...

```
BITPOS mykey 1 -100 -200
> 1
BITPOS mykey 1 -200 -100
> 1

BITCOUNT mykey -100 -200
> 0
BITCOUNT mykey -200 -100
> 4
```

BITCOUNT add a weird checking here...

Reply:
I suppose Redis docs would give some background?

Reply:
> I suppose Redis docs would give some background?

`-xx` means seek from backward. However, when `-xx` is greater than list size, Redis will pick the first byte.

So, `BITPOS mykey 1 -100 -200` is same as `BITPOS mykey 1 0 0` if mykey is more than 100 bytes.

However, BITCOUNT add a range check 😅, I don't understand the reason...